### PR TITLE
Added fetchFundingHistory to okex

### PR DESCRIPTION
**Pull request by samgermain**

### CODE REVIEW

File: js/okex.js [hunk 1]
```diff
@@ -1248,6 +1249,54 @@ module.exports = class okex extends Exchange {
...
+        //         ]
+        //     }
+        //
+        const rates = [];
+        const data = this.safeValue (response, 'data');
+        for (let i = 0; i < data.length; i++) {
+            const rate = data[i];
+            rates.push ({
+                'symbol': this.safeString (rate, 'instId'),
+                'fundingRate': this.safeNumber (rate, 'realizedRate'),
```
Comment (frosty00) [inline js/okex.js:1293]: you need to fill out all the keys, even if they are undefined

Comment (samgermain) [reply js/okex.js:1293]: All the exchanges only return these 3 properties

Comment (frosty00) [reply js/okex.js:1293]: ok, i would have expected that we had the datetime here as well, my bad

Comment (samgermain) [reply js/okex.js:1293]: > ok, i would have expected that we had the datetime here as well, my bad

I can add it, the method is new

Comment (kroitor) [reply js/okex.js:1293]: @samgermain adding the timestamp would be great, thanks!

Comment (samgermain) [reply js/okex.js:1294]: I added it, when I checked the other exchanges, I saw that it had already been changed so I thought it was done, but I forgot about this one

Comment (frosty00) [reply js/okex.js:1294]: thx!

yeah i added it to other exchanges in #10398

**Pull request merged**